Hardware/index.md

```diff
@@ -86,3 +90,21 @@ We support PC99-style Intel Architecture Platforms.
 | [PC99 (64-bit)](IA32) | x64  | VT-X           | VT-D  | [FC (without VT-X, VT-D and fastpath)][X64] | Data61         | seL4 Foundation        |
 
 [X64]: /projects/sel4/verified-configurations.html#x64
+
+
+---
+
+## *Unmaintained platforms*
+
+
+### *ARM*
+
+
+| *Platform* | *System-on-chip* | *Core* | *Arch* | *Virtualisation* | *SMMU* | *Verification Status* | *Contributed by* |
```

Comment:
We should add an explanation what unmaintained means and what people can expect from that. Maybe

> Unmaintained platforms are platforms for which code has been contributed, but this code is not or no longer tested and is unlikely to work. We list these here, because bringing an unmaintained platform up may be faster and easier than starting from scratch on a new platform port.

Reply:
added. thanks!